tests/v2/test_waku_filter.nim

```diff
@@ -18,3 +26,54 @@ procSuite "Waku Filter":
     check:
       decode.isErr == false
       decode.value == rpc
+
+  asyncTest "handle filter":
+    let
+      proto = WakuFilter.init()
```

Comment:
Are we sending WakuFilter over the wire? I thought the only thing we are sending over the wire here is related to ContentFilter, or possibly some RPC.

If we aren't sending it over the wire, why is this a protobuf?